components/brave_wallet/browser/polkadot/polkadot_substrate_rpc_responses.idl

```diff
@@ -39,4 +39,19 @@ namespace polkadot_substrate_rpc_responses {
     DOMString? result;
     Error? error;
   };
+
+  dictionary ChainHeader {
```

Comment:
Please, document dictionaries and fields. See https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/video_rvfc/video_frame_callback_metadata.idl